apps/mark-scan/frontend/src/pages/pat_device_identification/pat_device_identification_page.tsx

```diff
@@ -0,0 +1,65 @@
+import { Main, Screen, P, Font, Button } from '@votingworks/ui';
+import { useCallback, useEffect, useState } from 'react';
+import {
+  DiagnosticScreenHeader,
+  StepContainer,
+} from '../diagnostic_screen_components';
+import { handleGamepadKeyboardEvent } from '../../lib/gamepad';
+import { PatIntroductionStep } from './pat_introduction_step';
+import { IdentifyInputStep } from './identify_input_step';
+
+interface Props {
+  onAllInputsIdentified: () => void;
+  onExitCalibration: () => void;
+}
+
+export function PatDeviceIdentificationPage({
+  onAllInputsIdentified,
+  onExitCalibration,
+}: Props): JSX.Element {
+  const [step, setStep] = useState(0);
+
+  const nextStep = useCallback(() => {
+    setStep(step + 1);
+  }, [step, setStep]);
+
+  useEffect(() => {
+    // Child components override the gamepad keyboard listener while we take
+    // the voter through device identification.
+    document.removeEventListener('keydown', handleGamepadKeyboardEvent);
+
+    // Reattach the gamepad listener on cleanup
+    return () => {
+      document.addEventListener('keydown', handleGamepadKeyboardEvent);
+    };
+  });
```

Comment:
I wanted to prevent sip/puff events during the calibration from actually navigating and selecting elements, but in retrospect think `e.preventDefault` also does this. I'll confirm and remove this.

Reply:
https://github.com/votingworks/vxsuite/pull/3997/commits/ec3ea46da5d208dc7606afa9f645014e54dfddf2#diff-9667732ed6e98f6d65fe7f4b3a79f59dcb33cc61b08060a44ff3679ce5fee507L26